src/olympia/pages/urls.py

```diff
@@ -48,7 +48,7 @@
         name='pages.sunbird'),
     url('^sunbird/', lambda r: perma_redirect(reverse('pages.sunbird'))),
 
-    url('^shield-study-1/',
```

Comment:
All previous study links are wrapped up and I can't imagine anyone of the users we surveyed bookmarked it.  Should be fine from my perspective.